src/Extensions/DefaultRecordsExtension.php

```diff
@@ -24,9 +28,11 @@ public function requireDefaultRecords(): void
         // So provided SiteConfig doesn't exist, we know this is the first dev/build on a fresh DB,
         // and we want to create default pages, menus and blocks.
         if (!SiteConfig::get()->first()) {
+            // Create Home page
+            $this->createHomePage();
+
             // Create pages (adding pages here interferes with the core SiteTree logic which will create the About and
-            // Contact pages based on the current page count, so we'll add our own ones here. Home page still gets made
-            // within the SiteTree requireDefaultRecords() hook).
+            // Contact pages based on the current page count, so we'll add our own ones here.
```

Comment:
Not related to your work - but do you think you can get this function to create a header button? I think a caveat with that is you need to directly instantiate a subclass of link when doing these programatically, eg. SiteTreeLink::class